website/docs/Babel.md

```diff
@@ -29,4 +29,4 @@ module.exports = {
 
 Once this is set up WebdriverIO will take care of the rest.
 
-Alternatively you can configure how @babel/register is run through the environment variables for [@babel/register](babel) or using wdio config's [autoCompileOpts section](configurationfile) .
+Alternatively you can configure how @babel/register is run through the environment variables for [@babel/register](https://babeljs.io/docs/babel-register) or using wdio config's [autoCompileOpts section](https://webdriver.io/docs/configuration) .
```

Comment:
We have to keep document ids so it works across different page translations. Can you try this one?

```suggestion
Alternatively you can configure how @babel/register is run through the environment variables for [@babel/register](https://babeljs.io/docs/babel-register) or using wdio config's [autoCompileOpts section](configurationfile#autocompileopts).
```